proto/raw-js-openapi/webpack.config.js

```diff
@@ -1,4 +1,16 @@
 const path = require('path');
+const aliases = {};
+for (var proto of ['application', 'config', 'console', 'hierarchicaltable', 'inputtable', 'object', 'partitionedtable', 'session', 'storage', 'table', 'ticket']) {
+    aliases[`../../deephaven/proto/${proto}_pb`] = `${__dirname}/src/shim/${proto}_pb`;
+    aliases[`../../deephaven/proto/${proto}_pb.js`] = `${__dirname}/src/shim/${proto}_pb`;
+    aliases[`deephaven/proto/${proto}_pb`] = `${__dirname}/src/shim/${proto}_pb`;
+    aliases[`real/${proto}_pb`] =  `${__dirname}/build/js-src/deephaven/proto/${proto}_pb`;
+}
+for (var proto of ['Flight', 'BrowserFlight']) {
+    aliases[`./${proto}_pb`] = `${__dirname}/src/shim/${proto}_pb`;
+    aliases[`${proto}_pb`] = `${__dirname}/src/shim/${proto}_pb`;
+    aliases[`real/${proto}_pb`] =  `${__dirname}/build/js-src/${proto}_pb`;
+}
```

Comment:
I might put the `real` aliases first as I _think_ webpack will stop at first match, so most specific should be first. But I'm not sure as we haven't used webpack in the web team for a while

Reply:
Ok - I've also changed the paths of the `real/` ones locally to skip the io/deephaven/proto dir. 

I'll add some comments here too, to explain what each line is needed for - the first one is needed because protobuf files can't import each other when using commonjs_strict(!?!?).